blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -517,8 +517,9 @@ def create_primitive(shape: str, **kwargs) -> "MeshObject":
         raise Exception("No such shape: " + shape)
```

Comment:
The documentation to this function is wrong. 
```python
"""
:param shape: The name of the primitive to create. Available: ["CUBE"]
:return:
"""
```

Should be: 
```python
"""
:param shape: The name of the primitive to create. Available: ["CUBE", "CYLINDER", "CONE", "PLANE", "SPHERE", "MONKEY"]
:return: The newly created MeshObject
"""
```

Reply:
Thanks! I will correct it :)

Reply:
Sorry to be so over correct, but you forgot the return value ;)

Reply:
No problem, fixed it. It's good to work thoroughly ;)